frontend/src/components/Shared/DashboardSideNavbar/DashboardSideNavbar.jsx

```
@@ -0,0 +1,229 @@
+
+      {/* Backdrop overlay for mobile menu */}
+      {isMobile && isOpen && (
+        <button 
+          className="fixed inset-0 bg-gray-900 bg-opacity-50 z-20"
+          onClick={() => setIsOpen(false)}
+          onKeyDown={(e) => {
+            if (e.key === 'Enter' || e.key === ' ') {
+              setIsOpen(false);
+            }
+          }}
+        />
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)